app/src/main/java/com/sun/imageeditor/utils/EditParameters.kt

```diff
@@ -1,8 +1,12 @@
 package com.sun.imageeditor.utils
 
 data class EditParameters(
-    val filterType: FilterType? = null,
-    val adjustType: Int? = null,
-    val brightness: Float? = null,
-    val contrast: Float? = null,
+    var filterType: FilterType? = null,
+    var adjustType: Int? = null,
+    var brightness: Float? = null,
+    var contrast: Float? = null,
+    // these properties for testing
+    var icon: Float? = null,
+    var crop: Float? = null,
+    var draw: Float? = null,
```

Comment:
```suggestion
    val filterType: FilterType? = null,
    val adjustType: Int? = null,
    val brightness: Float? = null,
    val contrast: Float? = null,
    // these properties for testing
    val icon: Float? = null,
    val crop: Float? = null,
    val draw: Float? = null,
```
cần thay đổi thì dùng copy nhá